CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ There are multiple ways to contribute:
 - Translating MapComplete to your own language can be done
   on [this website](https://hosted.weblate.org/projects/mapcomplete/)
 - If you encounter a bug, the [issue tracker](https://github.com/pietervdvn/MapComplete/issues) is the place to be
-- If you want to improve a theme, create a new theme, spot a typo in the repo... the best way is to open a pull request.
+- If you want to improve a theme, create a new theme, spot a typo in the repo... the best way is to open a pull request. Read more about [making your own theme](https://github.com/pietervdvn/MapComplete/blob/develop/Docs/Making_Your_Own_Theme.md).
```

Comment:
```suggestion
- If you want to improve a theme, create a new theme, spot a typo in the repo... the best way is to open a pull request. Read more about [making your own theme](/Docs/Making_Your_Own_Theme.md).
```

See https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file